packages/frontend-2/components/workspaces/Promo/Banner.vue

```diff
@@ -29,7 +29,7 @@
         <FormButton :disabled="!isAdmin" color="outline" @click="$emit('create')">
           Create workspace
         </FormButton>
-        <FormButton color="subtle">Learn more</FormButton>
+        <!-- <FormButton color="subtle">Learn more</FormButton> -->
```

Comment:
is this button not needed any more @andrewwallacespeckle ?

Reply:
There is nowhere for it to link to yet that I know of. Rather than have a button that does nothing, I commented it out until we clarify what this should do. I can uncomment and add a destination in another branch rather than block this one.